bootcamp-milestone-2/src/lib/getBlogs.ts

```diff
@@ -0,0 +1,17 @@
+import connectDB from "@/helpers/db";
+import IBlog from "../database/blogSchema";
+
+//fetches the blogs from the Database
+async function getBlogs() {
```

Comment:
You would definitely do this kind of thing if you wanted to fetch blogs in multiple places. Though since we are only doing it in one page, I feel it is fine to just define it above the Blog page component.

Reply:
gotcha! I've been rendering my blogs in the home page as well so I assume that makes it okay to redefine it?